README.md

```diff
@@ -1,54 +1,57 @@
+If the data are outside of the constraints, the module can either remove them, smooth them or keep them.
+The filter parameters are optional. If they are not provided, the module will not perform any filtering.
 
 ## Environment Variables
 
 ### Module Specific
 
 The following module configurations can be provided in a data service designer section on weeve platform:
 
-| Name                              | Environment Variables           | type   | Description                                              |
-| --------------------------------- | ------------------------------- | ------ | -------------------------------------------------------- |
-| Upper Threshold                   | UPPER_THRESHOLD                 | float  | Value above which data would be considered as anomaly    |
-| Lower Threshold                   | LOWER_THRESHOLD                 | float  | Value below which data would be considered as anomaly    |
-| Anomaly Positive Rate of Change   | ANOMALY_POSITIVE_RATE_OF_CHANGE | float  | Anomaly positive rate of change per second               |
-| Anomaly Negative Rate of Change   | ANOMALY_NEGATIVE_RATE_OF_CHANGE | float  | Anomaly negative rate of change per second               |
-| Out of Bound Data                 | OUT-OF-BOUND_DATA               | string | What to do with out of bound data: keep, remove, smooth  |
-| Input Label                       | INPUT_LABEL                     | string | The input label on which anomaly is detected             |
+| Environment Variables          | type   | Description                                                                                                      |
+| ------------------------------ | ------ | ---------------------------------------------------------------------------------------------------------------- |
+| UPPER_THRESHOLD                | float  | Any data points exceeding this threshold should be considered outliers.                                          |
+| LOWER_THRESHOLD                | float  | Any data points below this threshold should be considered outliers.                                              |
+| RATE_OF_CHANGE_UPPER_THRESHOLD | float  | Any data point where the rate of change (per second) is greater than this value should be considered an outlier. |
+| RATE_OF_CHANGE_LOWER_THRESHOLD | float  | Any data point where the rate of change (per second) is less than this value should be considered an outlier.    |
+| OUTLIER_POLICY                 | string | What to do with outliers: keep, remove, smooth                                                                   |
+| INPUT_DATA_LABEL               | string | The input label on which to detect outliers.                                                                     |
+| INPUT_TIME_LABEL               | string | JSON key for the timestamp. The timestamp should in epoch format (int).                                          |
```

Comment:
Is this a mandatory value? And if yes, do we have a module to add it in this format so this module can get the right input?

Reply:
yes, I was thinking the [timestamp](https://github.com/weeve-modules/timestamp) module